### DISCUSSION TITLE
Help! Forgot my password

### DISCUSSION THREAD

**Discussion by david-unwin** [akeeba/panopticon#365] [General]
I am having a 'senior' moment. I have forgotten my password to get into my Panopticon site and I can't see how to reset it ad there is no 'Forgotten Password' option. I am the only person with login credentials. 
Is there a file somewhere in the folders I can reset.
Thanks David

**Comment by nikosdion** [comment]
The password is stored hashed with bCrypt in the `#__users` table. Replace the contents of the `password` column (e.g. using phpMyAdmin) with `$2y$10$4i6.Ue4ciqkErypVwOSCjeR.4aiB9fYixfYNoFFbKP6jg8RSR0OHa`. This changes your user's password to `letmein`. Log into Panopticon and change your password.

There's a good reason password reset is not implemented: it's insecure. A password reset involving an email means that your access to the protected resource can be overridden if someone can intercept the email sent to you, or if someone has gained access to your email account. If someone uses a 128 character, randomly generated password for Panopticon and the easily guessable password `spike123` for their email address do you want to guess where that leaves them, security-wise? Yup. It's as if their Panopticon password was `spike123` as well. Not secure at all.

**Comment by david-unwin** [comment]
Hi Nicholas,
Thanks for that. Yes, I appreciate why password reset is not implemented, particularly if you are managing multiple sites.
Kind regards
David
